src/test/run-make-fulldeps/override-aliased-flags/Makefile

```diff
@@ -0,0 +1,22 @@
+-include ../tools.mk
+
+# FIXME: it would be good to check that it's actually the rightmost flags
+# that are used when multiple flags are specified, but I can't think of a
+# reliable way to check this.
```

Comment:
(cc @varkor for his comment on https://github.com/rust-lang/rust/pull/60426#discussion_r279932992 which I read just now)

Reply:
Can't we make `rustc` show or warn to `stderr` about which flag it chose? "Note: both foo and bar specified. Using bar", then, we could just capture its output and compare.